index.html

```diff
@@ -1,12 +1,270 @@
+        <h3 class="site-main-benefits-title visually-hidden">Гарантия</h3>
+        <input class="site-main-benefits-button" type=button value="Гарантия" action="">
+      </li>
+      <li class="site-main-benefits-item">
+        <h3 class="site-main-benefits-title visually-hidden">Кредит</h3>
+        <input class="site-main-benefits-button" type=button value="Кредит" action="">
+      </li>
+    </ul>
+    <div class="site-main-benefits-constructor">
+      <b class="site-main-benefits-title">Доставка</b>
+      <p class="site-main-benefits-content">Мы с удовольствием доставим ваш товар прямо
+        к вашему подъезду совершенно бесплатно! Ведь
+        мы неплохо заработаем, поднимая его на ваш этаж./p>
+      </div>
+  </section>
+
+  <section class="site-main-add-offers"></section>
+</main>
+
+
+<footer class="site-footer">
+  <section class="site-footer-about">
+    <hr class="site-footer-about">
+    <h2 class="site-footer-about-title">О нас</h2>
+    <p class="site-footer-about-content">Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.</p>
+    <p class="site-footer-about-content">Мы можем доставить ваш товар в самые отдалённые уголки России транспортными компаниями:</p>
+    <ul class="site-footer-about-list">
+      <li class="site-footer-about-item">Меловые линии</li>
+      <li class="site-footer-about-item">СДИК</li>
+      <li class="site-footer-about-item">ТоксБерри</li>
+    </ul>
+    <a class="site-footer-about-more" href="#">Подробнее о нас</a>
+  </section>
+
+  <section class="site-footer-contacts">
+    <hr class="site-footer-contacts">
+    <h2 class="site-footer-contacts-title">Контакты</h2>
+    <h3 class="site-footer-contacts-heading">Интернет-магазин «Девайс»</h3>
+    <p class="site-footer-contacts-content">Вы можете забрать товар сами, заехав в наш офис.<br>Заодно сможете проверить работоспособность покупки.</p>
+    <p class="site-footer-contacts-content">Санкт-Петербург, набережная реки Карповки, 5, литера П.<br>+7 (812) 812-12-12</p>
+    <h3 class="site-footer-contacts-heading">Время работы</h3>
+    <p class="site-footer-contacts-content">пн—пт: с 10:00 до 22:00<br>сб—вс: с 10:00 до 19:00</p>
+    <a class="site-footer-about-more" href="#">Напишите нам</a>
+  </section>
+
+  <section class="site-footer-subscription">
+    <h2 class="site-footer-subscription-title">Подпишитесь на рассылку</h2>
+    <p class="site-footer-subscription-subtitle">Еженедельный дайджест с обзорами новинок,
+      длительными тест-драйвами, распаковками, а также акции, спецпредложения и вот это вот всё.</p>
+
+    <form class="site-footer-subscription-form">
+      <input class="site-footer-subscription-input" type="email" placeholder="Ваш e-mail">
+      <input class="site-footer-subscription-button" type="submit" value="Подписаться">
+    </form>
+
+    <div class="site-footer-subscription-advantage">
+      <img src="img/checked.svg">
+      <p>Это бесплатно</p>
+    </div>
```

Comment:
```suggestion
      <p>Это бесплатно</p>
```

объясни почему я оставила только один параграф?

Reply:
Потому что декоративные изображения мы не вставляем тегами, используем CSS